README.md

```diff
@@ -58,7 +58,7 @@ USAGE:
    trash [global options] command [command options] [arguments...]
 
 VERSION:
-   v0.2.5
+   v0.3.0-dev
```

Comment:
Why this is change to v0.3.0-dev. If you download the latest release from github, i believe it is showing v.2.7

Reply:
Other than that, LGTM

Reply:
I've changed it to `v0.2.7`.

For me it was showing `v0.3.0-dev`, I probably built the master branch. The command line options are the same for both `v0.2.7` and `v0.3.0`.